app/controllers/groups_controller.rb

```diff
@@ -73,7 +72,9 @@ def rename_group
       params[:assignment_id] = @assignment.id
 
       if Grouping.exists?(assessment_id: @assignment.id, group_id: groupexist_id)
-        flash_now(:error, I18n.t('groups.group_name_already_in_use'))
+        flash.now[:error] = I18n.t('groups.group_name_already_in_use')
```

Comment:
This change is not appropriate; `flash_now` is the preferred helper method. Same on line 77.